test/test_infra.py

```diff
@@ -107,7 +107,7 @@ def _run_setup(self, *args):
             (sys.executable, 'setup.py') + args,
             stdout=subprocess.PIPE,
             check=True)
-        return proc.stdout.strip().decode("utf8")
+        return proc.stdout.strip().decode("utf8").replace("\r\n", "\n")
```

Comment:
What about passing `universal_newlines` in True?

Reply:
I was confused because `universal_newlines` is very definitely deprecated for `open`, but it seems in `subprocess` it does exactly what we want, here. I'll switch to that.